src/alert.h

```diff
@@ -102,7 +102,7 @@ class CAlert : public CUnsignedAlert
     bool RelayTo(CNode* pnode) const;
     bool CheckSignature(const std::vector<unsigned char>& alertKey) const;
     bool ProcessAlert(const std::vector<unsigned char>& alertKey, bool fThread = true); // fThread means run -alertnotify in a free-running thread
-    static void Notify(const std::string& strMessage, bool fThread);
+    static void NotifyInternal(const std::string& strMessage);
```

Comment:
I'm not sure what you mean, the difference between the Notify and the Notify Internal is that the latter also sends a message to the uiClient. Which is internal, not external...